OffchainLabs/arbitrum-token-bridge
# feat: Open tx history on cctp tab after transfer

### Summary

Open transaction history on the proper tab (cctp) after a deposit/withdrawal.



### Steps to test

Make a cctp transfer, tx history should open on CCTP tab, and with the proper network tab selected (To Goerli for withdrawals, To Arbitrum Goerli for deposits)
Selecting other tabs should still work and display transactions accordingly
